Lib/unittest/mock.py

```diff
@@ -2422,15 +2422,14 @@ def __set__(self, obj, val):
 
 
 def seal(mock):
-    """Disable the automatic generation of "submocks"
+    """Disable the automatic generation of child mocks.
 
     Given an input Mock, seals it to ensure no further mocks will be generated
     when accessing an attribute that was not already defined.
 
-    Submocks are defined as all mocks which were created DIRECTLY from the
-    parent. If a mock is assigned to an attribute of an existing mock,
-    it is not considered a submock.
-
+    The operation seals the mock passed in, all the mocks generated
+    by accessing an attribute of the mock argument and all assigned
+    mocks without a name or a `spec` recursively.
```

Comment:
Is this supposed to be RST or md? If it's RST I think it needs to be

```rst
``spec``
```

instead of:

```markdown
`spec`
```

I also find the sentence confusing and don't really understand what it means.

Reply:
I think you can re-word like this:

> The operation recursively seals the mock passed in, meaning that the mock itself, any mocks generated by accessing one of its attributes, and all assigned mocks without a name or `spec`.